code/game/gamemodes/gang/gang.dm

```diff
@@ -248,6 +249,19 @@ GLOBAL_LIST_INIT(gang_colors_pool, list("red","orange","yellow","green","blue","
 		gang_bosses += G.bosses
 	return gang_bosses
 
+/datum/game_mode/proc/shuttle_check()
+	if(forced_shuttle)
+		return
+	var/alive = 0
+	for(var/mob/living/L in GLOB.player_list)
```

Comment:
I figured if crew were playing Ghost roles and enjoying them, or even participating in the Gang War with them, that it should consider them as alive. I don't have a problem with just relying on the joined_player_list.